rpm/skopeo.spec

```diff
@@ -146,6 +145,10 @@ cp -pav systemtest/* %{buildroot}/%{_datadir}/%{name}/test/system/
 #define license tag if not already defined
 %{!?_licensedir:%global license %doc}
 
+# Include this to silence rpmlint.
+# Especially annoying if you use syntastic vim plugin.
+%check
```

Comment:
Would we consider at least a smoke-test, like `skopeo --version`? Anyway, that’s not this PR.